sklearn/utils/extmath.py

```diff
@@ -332,6 +333,12 @@ def randomized_svd(
             the value of `random_state` explicitly to suppress the deprecation
             warning.
 
+    lapack_svd_driver : {'gesdd', 'gesvd'}, default='gesdd'
```

Comment:
I would use double-quoted in the docstring to be consistent with the code (`black` does not check for docstring :))